apis/r/R/SOMACollectionBase.R

```diff
@@ -22,8 +22,14 @@ SOMACollectionBase <- R6::R6Class(
     },
 
     #' @description Add a new SOMA object to the collection. (lifecycle: experimental)
-    create = function() {
-      super$create()
+    #' @param internal_use_only Character value to signal 'permitted' call as
+    #' `new()` is considered internal and should not be called directly
```

Comment:
That may have been me in an earlier commit.  Here 'permitted' in (literally and figuratively) in quotes to state that the token value (a line or two below in the signature) is meant to signal 'use is permitted'.

But you make a valid point and it may be worth while to actually quote the expected token _explicitly_ and mention that it is just token.